HealthcareApp/src/Healthcare.Application/Appointments/Commands/Reject/RejectAppointmentCommand.cs

```diff
@@ -0,0 +1,5 @@
+using MediatR;
+
+namespace Healthcare.Application.Appointments.Commands.Reject;
+
+public record RejectAppointmentCommand(int AppointmentId) : IRequest<bool>;
```

Comment:
For business it would be great to specify the reason of rejection and the person who rejected it - it might probably be rejected by client, doctor or manually by administrator of the system.
Not mandatory comment, if it's out of the task scope - don't fix